src/__tests__/Typeahead-spec.js

```diff
@@ -0,0 +1,83 @@
+jest.unmock('../Typeahead');
+
+// Don't need to test these and they currently throw errors
+jest.setMock('ship-components-outsideclick', 'div');
+jest.setMock('ship-components-textinput', 'div');
+jest.setMock('react-addons-css-transition-group', 'div');
+
+import React from 'react';
+import TestUtils from 'react-addons-test-utils';
+
+describe('Typeahead', function(){
+  let Typeahead;
+
+  beforeEach(function() {
+    Typeahead = require('../Typeahead').default;
+  });
+
+  // Render without error
+  it('should render without error', function() {
+    let element = React.createElement(
+      Typeahead, // component class
+      {} // props go here
+    );
+
+    expect(() => TestUtils.renderIntoDocument(element))
+      .not.toThrow();
+  });
+
+  it('should exists', function() {
+    // Render into document
+    let typeahead = TestUtils.renderIntoDocument( <Typeahead/> );
+
+    expect(TestUtils.isCompositeComponent(typeahead)).toBeTruthy();
+  });
+
+  it('should have an <OutsideClick/> component', function() {
```

Comment:
I'm not sure how important it is to test if the component exists.  I think it would be more useful to test the functionality works. So simulate a click and see if it closes the list.